WPThemeReview/Tests/CoreFunctionality/ThemeTemplatesException/header.inc

```diff
@@ -0,0 +1,7 @@
+<!-- Annotation has to be on the second line as errors are thrown on line 1 and errors on annotation lines are ignored. -->
+// phpcs:set WPThemeReview.CoreFunctionality.PrefixAllGlobals prefixes[] my_theme
+<?php
+
```

Comment:
Why is the `$my_theme_var = 123; // OK, prefixed.` snippet missing here ?

Reply:
Forgot to add it, my bad.